Manual.md

```diff
@@ -127,14 +127,16 @@ exception, start a PR and make an argument for why that particular piece of
 software, while not meeting any of the following requirements, is a good candidate for
 the Void packages system.
 
-1. System: The software should be installed system-wide, not per-user.
+1. System: The software should be installed system-wide, and must not be per-user installable.
```

Comment:
"The software is best suited to system-wide, rather than per-user, installation."

Almost anything *can* be installed per-user, but whether it should be is another matter. This requirement will always be nebulous; even things like fonts or themes become "suited" to system-wide installation in a multi-user system where more than one user prefers the font or theme. Words like "best suited" seem to strengthen this requirement a bit, without attempting too precise a restriction that will invite arguments when some packages are rejected.

Reply:
Makes sense to me - PR updated.